app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  after_save :updates_post_counter
+
+  def updates_post_counter
+    user.increment!(:postsCounter)
+  end
+
+  def recent_comments
+    comments.limit(5).order(created_at: :desc)
```

Comment:
- [ ] [Optional] Although this is very fine, but the project requirements specify that you should return the most recent 3 posts instead of 5
```suggestion
    comments.limit(3).order(created_at: :desc)
```